### PR TITLE
Add insertIds by default.

### PR DESCRIPTION
**Pull request by qingling128**
This depends on https://github.com/GoogleCloudPlatform/google-fluentd/pull/111.

### CODE REVIEW

File: templates/etc/td-agent/td-agent.conf [hunk 1]
```diff
@@ -20,6 +20,12 @@
   @type null
 </match>
 
+# Add a unique insertId to each log entry. This helps guarantee the order and
```
Comment (igorpeshansky) [inline templates/etc/td-agent/td-agent.conf]: `to each log entry that doesn't already have it`, right?

Comment (qingling128) [reply templates/etc/td-agent/td-agent.conf]: Yup! Changed.

File: templates/etc/td-agent/td-agent.conf [hunk 2]
```diff
@@ -20,6 +20,12 @@
   @type null
 </match>
 
+# Add a unique insertId to each log entry that doesn't already have it. This
```
Comment (igorpeshansky) [inline templates/etc/td-agent/td-agent.conf]: [Optional] At this point, I'd move the `This` to the next line to keep the sentence together. Ditto below.

Comment (qingling128) [reply templates/etc/td-agent/td-agent.conf]: Done

**Pull request merged**